pkg/container/container.go

```diff
@@ -62,6 +71,21 @@ type Container interface {
 	// SelinuxLabel returns the container's SelinuxLabel
 	// it takes the sandbox's label, which it falls back upon
 	SelinuxLabel(string) ([]string, error)
+
+	// spec functions
+
+	// returns the spec
+	Spec() *generate.Generator
+
+	// SpecAddMount adds a mount to the container's spec
+	// it takes the destination, the mount type, the source type, the mount to remove, and the options
+	SpecAddMount(string, string, string, string, []string)
+
+	// SpecAddAnnotations adds annotations to the spec.
+	// it takes the sandbox, the list of container volumes, the mount path
+	// the configStopSignal, an image result, a bool if it is systemd,
+	// and a bool systemdHasCollectMode
+	SpecAddAnnotations(*sandbox.Sandbox, []oci.ContainerVolume, string, string, *storage.ImageResult, bool, bool) error
```

Comment:
same here, these "string, string" and "bool, bool" are crying to be named.

Reply:
fixed!